catalog/from-k8s/resource.go

```diff
@@ -43,6 +43,10 @@ type ServiceResource struct {
 	// enabled (aka default enabled).
 	ExplicitEnable bool
 
+	// ClusterIPSync set to true (the default) syncs ClusterIP-type services.
```

Comment:
why default to true?

Reply:
This makes it easier for first time users to see catalog sync working since ClusterIP is the default service type.